core/snapshot/engine.go

```diff
@@ -439,14 +439,17 @@ func (e *Engine) applySnapshotFromLocalStore(ctx context.Context) error {
 }
 
 func (e *Engine) ReceiveSnapshot(snap *types.Snapshot) error {
+	if e.Config.StartHeight > 0 && snap.Height != uint64(e.Config.StartHeight) {
```

Comment:
wouldn't snap.Height <= uint64(e.Config.StartHeight) suffice?

Reply:
Yea it would, but the same config is used for loading from local snapshots and that looks for height equality. I have no preference, I can change both to be `<=` if that seems more useful?

Reply:
I don't mind, just wanted to make sure I didn't misunderstand this. It's awesome that we can do this.